applications/typechain_revived.md

```diff
@@ -0,0 +1,170 @@
+# Improve & Update Typechain Polkadot
+
+- **Team Name:** C Forge
+- **Payment Address:** 15VHeCpx5SUbRsbiZAfYtERxXPNKNZfc9bNBnGGZ4SvaT2ct (USDT)
```

Comment:
Please indicate whether you want this address to be used for DOT payments on Polkadot as well, or a separate address.

Reply:
Hi, sorry, I thought it was implied by PRs description checkbox. I did put a notice in the PR itself now.

Reply:
Ah, sorry, that's actually outdated as we transfer DOT on Polkadot directly, not AssetHub.